macrofusion.py

```diff
@@ -611,17 +644,40 @@ def preview(self, widget):
 
     def update_align_options(self):
         if self.checkbutton_a5_align.get_active():
-            settings["align_settings"]["auto_crop"][1]     = self.checkbutton_a5_crop.get_active()
-            settings["align_settings"]["opt_img_shift"][1] = self.checkbutton_a5_shift.get_active()
-            settings["align_settings"]["opt_fov"][1]       = self.checkbutton_a5_field.get_active()
+            settings["align_settings"]["auto_crop"][1]          = self.checkbutton_a5_crop.get_active()
+            settings["align_settings"]["opt_img_shift"][1]      = self.checkbutton_a5_shift.get_active()
+            settings["align_settings"]["opt_fov"][1]            = self.checkbutton_a5_field.get_active()
+            settings["align_settings"]["opt_radial_dist"][1]    = self.checkbutton_a5_radial.get_active()
+        
+        if self.check_corrthres.get_active():
+            settings["align_settings"]["corr_thres"][1]       = self.spinbuttoncorrthres.get_value()
+        else:
+            settings["align_settings"]["corr_thres"][1]       = 0.0
+        
+        if self.check_pnt.get_active():
+            settings["align_settings"]["num_ctrl_pnt"][1]       = int(self.spinbuttonpnt.get_value())
+        else:
+            settings["align_settings"]["num_ctrl_pnt"][1]       = 0
+        
+        if self.check_pntthr.get_active():
+            settings["align_settings"]["ctrl_pnt_thr"][1]       = int(self.spinbuttonpntthr.get_value())
+        else:
+            settings["align_settings"]["ctrl_pnt_thr"][1]       = 0
+        
+        settings["align_settings"]["opt_x_coord"][1]       = self.check_xcoord.get_active()
+        settings["align_settings"]["opt_y_coord"][1]       = self.check_ycoord.get_active()
+        settings["align_settings"]["opt_z_coord"][1]       = self.check_zcoord.get_active()
 
     def update_enfuse_options(self):
         settings["fuse_settings"]["exposure-weight"][1]     = self.spinbuttonexp.get_value()
         settings["fuse_settings"]["exposure-mu"][1]         = self.spinbuttonmu.get_value()
         settings["fuse_settings"]["exposure-sigma"][1]      = self.spinbuttonsigma.get_value()
         settings["fuse_settings"]["contrast-weight"][1]     = self.spinbuttoncont.get_value()
         settings["fuse_settings"]["saturation-weight"][1]   = self.spinbuttonsat.get_value()
-        settings["fuse_settings"]["levels"][1]              = self.spinbuttonlevel.get_value_as_int()
+        if self.check_pyramidelevel.get_active():
+            settings["fuse_settings"]["levels"][1]          = self.spinbuttonlevel.get_value_as_int()
+        else:
+            settings["fuse_settings"]["levels"][1]          = 0
```

Comment:
What was the motivation for setting the image pyramid levels to 0 if the checkbox is not active?
Quoting the [enfuse manual for this parameter](https://enblend.sourceforge.net/enfuse.doc/enfuse_4.2.xhtml/enfuse.html#opt:levels) here:

> If too few levels are used, “halos” around regions of strong local feature variation can show up. On the other hand, if too many levels are used, the image might contain too much global features. Usually, the latter is not a problem, but is highly desired. This is the reason, why the default is to use as many levels as is possible given the size of the overlap regions. Enfuse may still use a smaller number of levels if the geometry of the overlap region demands. 

E.g. when leaving it at 29 levels, I get enfuse output like that:
> enfuse: warning: cannot blend with 29 pyramid level(s) as
enfuse: warning: image geometry precludes using more than 8 pyramid level(s)

So I think it should be fine to leave it as it has been before, if there is no specific reason to change it.